.circleci/config.yml

```diff
@@ -311,7 +310,14 @@ jobs:
          pr-name: << pipeline.parameters.pr_number >>
       - pr_comment:
          when: on_success
-         comment: "Deployed prnet successfully"
+         comment: "\
+           <p>PR NET pr-<< pipeline.parameters.pr_number >> Successfully deployed!</p>\ 
+           <p>explorer=https://explorer-pr-<< pipeline.parameters.pr_number >> </p> \
```

Comment:
We are missing the last part of the domain no? 
I saw on the previous comments that the link was clickable, I think it would be best if it was possible to click.
like: https://explorer-pr-1023.prnet.taraxa.io